tests/virtualization/universal/prepare_guests.pm

```diff
@@ -20,6 +20,12 @@ sub run {
     my $self = shift;
     # Use serial terminal, unless defined otherwise. The unless will go away once we are certain this is stable
     $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    # Clean up VMs if any. This makes it possable to reinstall VMs.
+    systemctl("restart libvirtd");
+    assert_script_run('for i in $(virsh list --name|grep sles);do virsh shutdown $i;done');
```

Comment:
"Virsh destroy" can force the guest off like physically power off and is faster than 'virsh shutdown'. IMHO, 'virsh shutdown' can be removed.

Reply:
Yes, you are right. "Virsh destroy" is sufficient.